core/blockly.js

```diff
@@ -261,13 +330,110 @@ Object.defineProperties(exports, {
       common.setSelected(newSelection);
     }
   },
+  ALIGN_LEFT: {
```

Comment:
In #5567 @cpcallen has started adding a jsdoc above these so that they will have the appropriate docs.